templates/eap72-amq-persistent-s2i.json

```diff
@@ -6,10 +6,10 @@
             "iconClass": "icon-eap",
```

Comment:
General note on this AMQ template, and the tx recovery one. We're currently pointing to "name": "amq-broker-71-openshift:1.0" but might be able to change that to use 7.2.

Reply:
@luck3y Is 7.2 the one that went GA this week?

Reply:
@bstansberry Yeah, but the latest info I have is its not going into OSO, so I guess we can bump to 7.2 but not have a template on OSO again...

Reply:
Thanks; I was just asking because I'm not doing well at keeping track of versions. :)

I'll add a commit to change that.

Reply:
I added a commit. I'll send a PR up for CD as well.